alphabot/environment.py

```diff
@@ -0,0 +1,40 @@
+"""Module for Environment abstract class"""
```

Comment:
Github provides a squash and merge option. Will use it after resolving conversations.

Reply:
We use Gitlab flow, hence we use rebase instead of squash & merge. That's why it's important to create a MR with already squashed commits. Btw don't resolve conversations on your own, author of the thread should do it to confirm that issue has been resolved.

Reply:
I feel like that doesn't matter. In [gitlab flow squashing commits](https://docs.gitlab.com/ee/topics/gitlab_flow.html#squashing-commits-with-rebase) there is also mentioned about Squash-and-merge feature, that allows reviewing commit history on a branch, in case of reverting the merge commit. In this particular case both approaches have the same effect (despite timestamps and additional commit on master branch). 
Altho I respect your request and will squash it manually right after all threads are resolved, to keep commit history clean.